src/ConnectedMode/Migration/VsAwareFileSystem.cs

```diff
@@ -29,26 +30,43 @@ namespace SonarLint.VisualStudio.ConnectedMode.Migration
     internal class VsAwareFileSystem : IVsAwareFileSystem
     {
         private readonly ILogger logger;
+        private readonly IFileSystem fileSystem;
 
         [ImportingConstructor]
-        public VsAwareFileSystem(ILogger logger) => this.logger = logger;
-        
+        public VsAwareFileSystem(ILogger logger)
+            : this(logger, new FileSystem())
+        { }
+
+        internal /* for testing */ VsAwareFileSystem(ILogger logger, IFileSystem fileSystem)
+        {
+            this.logger = logger;
+            this.fileSystem = fileSystem;
+        }
+
         public Task DeleteFolderAsync(string folderPath)
         {
-            // TODO
+            // TODO - error handling
+            LogVerbose($"Deleting directory: {folderPath}");
+            fileSystem.Directory.Delete(folderPath, true);
```

Comment:
Minimal implementation of all methods that just pass-through to the file system.